packages/wallet-connection/README.md

```diff
@@ -48,6 +48,27 @@ environment that `agoric-wallet-connection` uses:
 import './install-ses-lockdown.js';
 ```
 
+Or, in your `index.html`:
+
+```html
+<script src="lockdown.umd.js"></script>
+<script>
+  lockdown({
+    __allowUnsafeMonkeyPatching__: 'unsafe',
```

Comment:
Did you find that you needed unsafe monkey patching?

Reply:
For react, yes, just double-checked. However, I don't think in non-react it will always be required, so I just removed it from here and pointed it out in the react section.